### DISCUSSION TITLE
flask-migrate v4.04(latest) maybe happen some errors. it seem failed to create db shell command.

### DISCUSSION THREAD

**Discussion by simpleth** [miguelgrinberg/Flask-Migrate#511] [Q&A]
I failed to usage these bash commands:
$ flask db init
...
error shows flask maybe haven't the command

I don't know. also it is my mistake, I want to seek a revolution.

when I use V2.7,  it works well.

**Comment by miguelgrinberg** [comment]
I can't really tell you what's wrong, since you are not providing any details regarding the code that you are using. My recommendation is that you follow my Flask Mega-Tutorial to learn how to work with Flask, databases and Flask-Migrate.
